twitter1.css

```diff
@@ -201,8 +192,6 @@ body {
 .clr-grn {
   color: #74b816;
```

Comment:
Might be a good idea in future that whenever you add a new colour like here `color: #74b816;` you add it to your CSS vars like you have elsewhere :)